src/components/Tooltip/Tooltip.story.js

```diff
@@ -130,3 +131,38 @@ stories.add(
         </div>,
     ),
 );
+
+stories.add('Auto Position', () => (
+    <div>
+        <p>
+            Setting the `position` prop to `auto` or using `TooltipPosition.AUTO` allows the tooltip
```

Comment:
I think our enums are not exposed in the library exports. They are probably still accessible using direct path imports but it's better not to suggest it in the docs.

Reply:
I wasn't sure when I wrote this, thanks for confirming